src/base/gfx/colortransform.cpp

```diff
@@ -73,6 +73,15 @@ ColorTransform ColorTransform::Opacity(double factor)
 	    "opacity(" + std::to_string(factor) + ")"};
 }
 
+ColorTransform ColorTransform::Fade(double factor)
```

Comment:
We should call this opacity, because this is what CSS opacity does. What is now opacity directly sets the alpha channel, so we should call that alpha instead.